src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,273 @@
+	 * Get the name under which to register the CLI command.
+	 *
+	 * @return string The name under which to register the CLI command.
+	 */
+	public static function get_command_name() {
+		return 'amp option';
+	}
+
+	/**
+	 * OptionCommand constructor.
+	 *
+	 * @param ReaderThemes      $reader_themes ReaderThemes instance.
+	 * @param PluginSuppression $plugin_suppression PluginSuppression instance.
+	 */
+	public function __construct( ReaderThemes $reader_themes, PluginSuppression $plugin_suppression ) {
+		$this->reader_themes      = $reader_themes;
+		$this->plugin_suppression = $plugin_suppression;
+	}
+
+	/**
+	 * Gets the value for an option.
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 */
+	public function get( $args, $assoc_args ) {
+		list( $option_name ) = $args;
+
+		$this->check_user();
+
+		$options = $this->get_options();
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Option %s does not exist.', 'amp' ), $option_name ), false );
+			WP_CLI::line( WP_CLI::colorize( '%y' . __( 'Try using `wp amp option list` to see all available options.', 'amp' ) . '%n' ) );
+			WP_CLI::halt( 1 );
+		}
+
+		WP_CLI\Utils\format_items(
+			'table',
+			[
+				'option_name'  => $option_name,
+				'option_value' => $options[ $option_name ],
+			],
+			[ 'option_name', 'option_value' ]
+		);
+	}
+
+	/**
+	 * Updates the value for an option.
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 *
+	 * @alias set
+	 */
+	public function update( $args, $assoc_args ) {
+		list( $option_name, $option_value ) = $args;
+
+		$help_message = __( 'Try using `wp amp option list` to see all available options.', 'amp' );
+
+		// Check if option is allowed to be managed via CLI.
+		if ( ! in_array( $option_name, self::ALLOWED_OPTIONS, true ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Option %s is not allowed to be managed via CLI.', 'amp' ), $option_name ), false );
+			WP_CLI::line( WP_CLI::colorize( '%y' . $help_message . '%n' ) );
+			WP_CLI::halt( 1 );
+		}
+
+		$this->check_user();
+
+		$options = $this->get_options();
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Option %s does not exist.', 'amp' ), $option_name ), false );
+			WP_CLI::line( WP_CLI::colorize( '%y' . $help_message . '%n' ) );
+			WP_CLI::halt( 1 );
+		}
+
+		// Update the option.
+		$this->update_option( $option_name, $option_value );
+	}
+
+	/**
+	 * List AMP options.
+	 *
+	 * @subcommand list
+	 */
+	public function list_() {
+		$this->check_user();
+
+		$options = $this->get_options();
+
+		// Add reader themes to the options.
+		$options[ self::READER_THEMES ] = wp_list_pluck( $this->reader_themes->get_themes(), 'slug' );
```

Comment:
This may be confusing to a user as this is not actually an option.

If we want a way to list the available Reader themes, this can be done via a separate command entirely. For example, `wp amp options list-reader-themes`.